pandas/io/formats/excel.py

```diff
@@ -76,15 +81,23 @@ def __call__(self, declarations_str: str):
 
         Returns
         -------
-        xlstyle : dict
-            A style as interpreted by ExcelWriter when found in
-            ExcelCell.style.
+        Dict
+            A style as interpreted by ExcelWriter when found in ExcelCell.style.
         """
         # TODO: memoize?
         properties = self.compute_css(declarations_str, self.inherited)
         return self.build_xlstyle(properties)
 
-    def build_xlstyle(self, props):
+    def build_xlstyle(self, props: Dict) -> Dict[str, Dict]:
```

Comment:
Can you add subtypes for props? I think just `Dict[str, str]` and can maybe annotate `CSSResolver` as part of this